### DISCUSSION TITLE
telemetry is reset

### DISCUSSION THREAD

**Discussion by Benno00** [FrSkyRC/ETHOS-Feedback-Community#4167] [General]
Hello,
is there an event when the telemetry is reset?
I have the problem that i can display min. / max. values ​​in my COLORWIDGET, it always shows the smallest or highest value reached. But when the telemetry is reset, I don't notice it. Or can you read the current min. / max. value using LUA, unfortunately I haven't found anything about that.

Many thanks
Regards Benno

**Comment by bsongis-frsky** [comment]
The constant `SYSTEM_EVENT_FLIGHT_RESET` will be exported to the Lua interface with 1.5.14
